src/main/java/org/jenkinsci/plugins/codesonar/conditions/WarningCountIncreaseSpecifiedScoreAndHigherCondition.java

```diff
@@ -1,32 +1,35 @@
 package org.jenkinsci.plugins.codesonar.conditions;
 
-import hudson.Extension;
-import hudson.Launcher;
-import hudson.model.Run;
-import hudson.model.TaskListener;
-import hudson.model.Result;
-
 import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.annotation.Nonnull;
 
-import hudson.util.FormValidation;
 import org.apache.commons.lang.StringUtils;
 import org.jenkinsci.Symbol;
-import org.jenkinsci.plugins.codesonar.CodeSonarBuildAction;
+import org.jenkinsci.plugins.codesonar.CodeSonarLogger;
 import org.jenkinsci.plugins.codesonar.models.CodeSonarBuildActionDTO;
 import org.jenkinsci.plugins.codesonar.models.analysis.Analysis;
 import org.jenkinsci.plugins.codesonar.models.analysis.Warning;
 import org.kohsuke.stapler.DataBoundConstructor;
 import org.kohsuke.stapler.DataBoundSetter;
 import org.kohsuke.stapler.QueryParameter;
 
-import javax.annotation.Nonnull;
+import hudson.Extension;
+import hudson.Launcher;
+import hudson.model.Result;
+import hudson.model.TaskListener;
+import hudson.util.FormValidation;
 
 /**
  * @author andrius
  */
 public class WarningCountIncreaseSpecifiedScoreAndHigherCondition extends Condition {
+    private static final Logger LOGGER = Logger.getLogger(WarningCountIncreaseSpecifiedScoreAndHigherCondition.class.getName());
 
     private static final String NAME = "Warning count increase: specified score and higher";
+    private static final String RESULT_DESCRIPTION_MESSAGE_FORMAT = "score={0,number,0} threshold={1,number,0.00}% increase={2,number,0.00}%";
```

Comment:
For consistency, include commas between items, e.g. "score=60, threshold=1.00%, increase=0.75%".